backend/.github/workflows/validate-pr-title.yml

```diff
@@ -0,0 +1,14 @@
+# This workflow validates pull request titles against Commitlint rules.
+name: Validate PR Title
+
+on:
+  pull_request:
+
+jobs:
+  validate-title:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: dreampulse/action-lint-pull-request-title@v3 # Action for title validation
+        with:
+          config: '.commitlintrc.js'
```

Comment:
I don't think we currently have a .commitlintrc.js file.

Try this once: https://github.com/keyshade-xyz/keyshade/blob/develop/.github/workflows/pr-lint.yaml

Reply:
If you want, then do go ahead with it!

Reply:
Cool, looks good in that case!